src/HookFactory.php

```diff
@@ -44,7 +44,11 @@ function %%function_name%% {
     RESULT="$(%%completion_command%% </dev/null)";
     STATUS=$?;
 
-    local cur;
+    local cur mail_check_backup;
+
+    mail_check_backup=$MAILCHECK
+    export MAILCHECK=0
```

Comment:
0 is *greater than or equal to zero* though. A value of zero causes the mail check to occur without any delay. It needs to be set to -1 or use `unset MAILCHECK`. Sorry, saw that was wrong the other day and didn't have a chance to mention it.